instagram-app/src/components/PostContainer/AddComment.js

```diff
@@ -1,19 +1,37 @@
 import React from 'react'
 import PropTypes from 'prop-types';
-import './PostContainer.css'
-import {Input, InputGroup, InputGroupAddon, Button} from 'reactstrap'
+import './PostContainer.css';
+import styled from 'styled-components';
 
 const AddComments = (props)=>{
+    const AddComments = styled.div`
```

Comment:
Nice job adding styled components. This is not the right way to do, and indeed it's leading to some major bugs:  your input fields are not working as intended.
You should declare the styledComponent outside of the actual component and the passing it in the render function. 
You could want to refactor this.